packages/components/src/SegmentedControl/src/SegmentedControlItem.module.css

```diff
@@ -0,0 +1,121 @@
+.hop-SegmentedControlItem {
```

Comment:
We're really now in the fine tuning section, but when focusing the segmented control with the keyboard, the outline is barely visible. Now that it's now showing on click, it's really ok to have a big blue outline like the buttons have.